backend/src/index.ts

```diff
@@ -1,64 +1,59 @@
 import express, { Express, NextFunction, Request, Response } from 'express';
-
+import path from 'path';
 import mongoose from 'mongoose';
 import cors from 'cors';
 import dotenv from 'dotenv';
 import cookieParser from 'cookie-parser';
 
 import authRoutes from './apis/auth/routes';
 import contactRoutes from './apis/contacts/routes';
-import { ErrorWithStatus } from './global/interfaces/ErrorWithStatus';
+import { IError } from './lib/interfaces/IError';
+import DateHelper from './lib/helpers/DateHelper';
+import { jsonApiErrorResponseFromError } from './lib/helpers/jsonApiErrorResponseFromError';
 
 dotenv.config();
 
 const app: Express = express();
-
-//mongodb atlas node 2.2.12
 const MONGODB_URI = `mongodb://${process.env.MONGODB_USER}:${process.env.MONGODB_PASSWORD}@ac-yztvzc4-shard-00-00.b5tvnqi.mongodb.net:27017,ac-yztvzc4-shard-00-01.b5tvnqi.mongodb.net:27017,ac-yztvzc4-shard-00-02.b5tvnqi.mongodb.net:27017/?ssl=true&replicaSet=atlas-dbcw9j-shard-0&authSource=admin&retryWrites=true&w=majority`;
```

Comment:
If `MONGODB_USER` and `MONGODB_USER` are always used in `MONGODB_URI`, I would suggest using `MONGODB_URI` directly as the environment variable. Currently I'm not seeing how you can point to different environments either; which is obviously a big consideration for an _environment_ variable.

Reply:
done. combined MONGODB_USER and MONGODB_PASSWORD as MONGODB_URI